classes/migration/upgrade/v3_4_0/PreflightCheckMigration.php

```diff
@@ -288,14 +290,13 @@ protected function getDataFromLogEntry(string $entry): array
         // The default regex that can parse the usageStats plugin's log files.
         $parseRegex = '/^(?P<ip>\S+) \S+ \S+ "(?P<date>.*?)" (?P<url>\S+) (?P<returnCode>\S+) "(?P<userAgent>.*?)"/';
         if (preg_match($parseRegex, $entry, $m)) {
-            $associative = count(array_filter(array_keys($m), 'is_string')) > 0;
-            $entryData['ip'] = $associative ? $m['ip'] : $m[1];
-            $time = $associative ? $m['date'] : $m[2];
+            $entryData['ip'] = $m[1];
```

Comment:
Oh, I personally like the named groups (you don't need to update indexes if something changes), as you're not using them, it's possible to remove the `?P<identifier>` from the regexp. But I personally would leave (it also works as a kind of documentation).

Reply:
Ah, I understood it the other way around -- that numbered indexes are better :sweat_smile: